[2023-09-28T09:06:14.212+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: user_processing.create_table scheduled__2023-09-27T00:00:00+00:00 [queued]>
[2023-09-28T09:06:14.218+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: user_processing.create_table scheduled__2023-09-27T00:00:00+00:00 [queued]>
[2023-09-28T09:06:14.219+0000] {taskinstance.py:1359} INFO - Starting attempt 2 of 2
[2023-09-28T09:06:14.230+0000] {taskinstance.py:1380} INFO - Executing <Task(PostgresOperator): create_table> on 2023-09-27 00:00:00+00:00
[2023-09-28T09:06:14.236+0000] {standard_task_runner.py:57} INFO - Started process 97 to run task
[2023-09-28T09:06:14.240+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'scheduled__2023-09-27T00:00:00+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp947k9htq']
[2023-09-28T09:06:14.241+0000] {standard_task_runner.py:85} INFO - Job 32: Subtask create_table
[2023-09-28T09:06:14.275+0000] {task_command.py:415} INFO - Running <TaskInstance: user_processing.create_table scheduled__2023-09-27T00:00:00+00:00 [running]> on host 248791025d92
[2023-09-28T09:06:14.322+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='user_processing' AIRFLOW_CTX_TASK_ID='create_table' AIRFLOW_CTX_EXECUTION_DATE='2023-09-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-09-27T00:00:00+00:00'
[2023-09-28T09:06:14.324+0000] {sql.py:274} INFO - Executing: 
            CREATE TABLE IF NOT EXISTS users (
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            );
        
[2023-09-28T09:06:14.330+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2023-09-28T09:06:14.333+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2023-09-28T09:06:14.335+0000] {sql.py:418} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            );
        , parameters: None
[2023-09-28T09:06:14.340+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20230927T000000, start_date=20230928T090614, end_date=20230928T090614
[2023-09-28T09:06:14.372+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-28T09:06:14.388+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
